docs/dev/isolation.md

```diff
@@ -23,7 +23,7 @@ The groups currently defined are:
 | gossip                                       | 1000
 | streaming (a.k.a maintenance)                | 200
 
-TODO: explain the purpose each of each of these scheduling groups, and what they are used for. E.g., "streaming" is also called maintenance and used also used for repair. memtable is used for memtable flushes (?). default is used for gossip, etc.
+TODO: explain the purpose each of these scheduling groups, and what they are used for. E.g., "streaming" is also called maintenance and used also used for repair. memtable is used for memtable flushes (?). default is used for gossip, etc.
```

Comment:
The fix is not correct - the first word "each" was indeed repeated and should be deleted, but the extra "of" is necessary - we want "explain the purpose of each of".